scripts/parallel_preprocess_trajectories.py

```diff
@@ -65,6 +64,7 @@ def write_images_to_lmdb(mp_arg):
                     f"{idx}".encode("ascii"), pickle.dumps(do, protocol=-1)
                 )
                 txn.commit()
+                sampled_unique_ids += process_samples[i]
```

Comment:
Just making sure — you ran and checked that it works, right?

My impression was that if `process_samples[i]` is an int, `+=` with a list doesn't work.

Reply:
`sampled_unique_ids.append(process_samples[i])` would work

Reply:
Oops this slipped me. You're too good.